package.json

```diff
@@ -12,17 +12,21 @@
     "test-docker": "docker run blip:test",
     "test": "TZ=UTC NODE_ENV=test karma --log-level debug start",
     "test-watch": "TZ=UTC NODE_ENV=test karma start --no-single-run --reporters=mocha",
-    "test-cloudfront": "bash test/lambda/run-lambda-tests.sh",
+    "test-cloudfront": "TARGET_ENVIRONMENT=test API_HOST='https://api.example.com' node cloudfront-dist/cloudfront-gen-lambda.js && mocha test/lambda/cloudfront-gen-lambda.test.js",
     "start-dev": "NODE_ENV=development USE_WEBPACK_DEV_SERVER=true ALLOW_SIGNUP_PATIENT=false ALLOW_PATIENT_CHANGE_EMAIL=false I18N_ENABLED=true ALLOW_PATIENT_CHANGE_PASSWORD=false HIDE_DONATE=true HIDE_DEXCOM_BANNER=true HIDE_UPLOAD_LINK=true CAN_SEE_PWD_LOGIN=true BRANDING='diabeloop' METRICS_SERVICE='disabled' HELP_LINK='disabled' webpack-dev-server --cache --colors --progress --port 3001 --host 0.0.0.0",
     "prebuild": "rm -rf dist server/dist deploy",
-    "build": "npm run build-app && npm run build-config",
+    "build": "npm run build-app && TARGET_ENVIRONMENT=test npm run gen-lambda",
+    "pregen-lambda": "NODE_ENV=production npm install",
+    "gen-lambda": "BLIP_VERSION=$(echo \"const c = require('./package.json'); console.log(c.version);\" | node -) node cloudfront-dist/cloudfront-gen-lambda.js",
+    "deploy-cf": "echo 'not implemented yet'",
     "prebuild-app": "echo 'Building app'",
-    "build-app": "NODE_ENV=production webpack",
+    "build-app": "NODE_ENV=production webpack && mv static-dist/index.html cloudfront-dist/pre-compiled-index.html",
```

Comment:
I wouldn't have put generated files in the cloudfront-dist directory

Reply:
why not?

Reply:
I think it is cleaner to have all generated file put in one directory if possible.
It's easier to clean up the build too, and less confused when you need to do partial build.